move-vm-backend/tests/assets/move-projects/substrate_balance/Move.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "substrate_balance"
+version = "0.0.0"
+
+[dependencies]
+MoveStdlib = { git = "https://github.com/eigerco/substrate-move.git", subdir = "language/move-stdlib", rev = "main" }
```

Comment:
Maybe we could stick to one naming scheme - if we have `substrate-stdlib` then we may use `move-stdlib` instead of `MoveStdLib`.
But if it's a convention used across other native Move modules, then leave it as is.

Reply:
Yeah, the `move-stdlib` would be a more preferred name, but we can change it maybe in the future before we move `move-stdlib` Move source files from this repo to a specific repo just for that purpose.

But that change will have 500+ changes across our repo since MoveStdlib is used everywhere (in tests and all Move projects) - so I wasn't sure what to do here. Just the renaming should be done in a separate PR probably :D

Reply:
So let's leave it as it is for now.